var/spack/repos/builtin/packages/bamutil/package.py

```diff
@@ -14,23 +14,32 @@ class Bamutil(MakefilePackage):
 
     homepage = "https://genome.sph.umich.edu/wiki/BamUtil"
     url      = "https://genome.sph.umich.edu/w/images/7/70/BamUtilLibStatGen.1.0.13.tgz"
+    git      = "https://github.com/statgen/bamUtil.git"
+    maintainers = ['snehring']
 
+    version('1.0.15', commit='3ad3980a3a3a3fc35eca3636b7206676c8303ce6')
     version('1.0.13', sha256='16c1d01c37d1f98b98c144f3dd0fda6068c1902f06bd0989f36ce425eb0c592b')
 
-    depends_on('zlib', type=('build', 'link'))
+    depends_on('zlib')
+    depends_on('git', type='build', when='@1.0.15:')
```

Comment:
Is there a use case where this is needed (and not already in the builder's path)?

Reply:
If someone were building in an environment that doesn't include git somehow, like in a container to mimic a buildhost.